redis/src/test/scala/zio/redis/SortedSetsSpec.scala

```diff
@@ -986,6 +986,27 @@ trait SortedSetsSpec extends BaseSpec {
           } yield assert(result)(isNone)
         }
       ),
+      suite("zMScore")(
+        testM("non-empty set") {
+          for {
+            key <- uuid
+            _ <- zAdd(key)(
+                   MemberScore(10d, "Delhi"),
+                   MemberScore(20d, "Mumbai"),
+                   MemberScore(30d, "Hyderabad"),
+                   MemberScore(40d, "Kolkata"),
+                   MemberScore(50d, "Chennai")
+                 )
+            result <- zMScore(key, "Delhi", "Mumbai")
+          } yield assert(result)(equalTo(Chunk(Some(10d), Some(20d))))
```

Comment:
I'd include the "non-existing" value here.

Reply:
good case 👍